src/app/components/ContactsList.js

```diff
@@ -157,6 +156,8 @@ const ContactsList = ({ contacts, onCheck, history, contactID, location }) => {
 ContactsList.propTypes = {
     contacts: PropTypes.array,
     onCheck: PropTypes.func,
+    hasPaidMail: PropTypes.number,
```

Comment:
I prefer if you pass `user` in case `hasPaidMail` change and becomes a `Boolean`.

Reply:
That's what I was doing at the beginning, but then I saw that in other files. Will try to change it everywhere since a Boolean looks nicer indeed